searchcore/src/vespa/searchcore/config/proton.def

```diff
@@ -213,20 +213,22 @@ summary.cache.initialentries long default=0 restart
 summary.cache.compression.type enum {NONE, LZ4, ZSTD} default=LZ4 restart
 
 ## Control compression level of the summary while in cache.
+## LZ4 has normal range 1..9 while ZSTD has range 1..19
 summary.cache.compression.level int default=9 restart
```

Comment:
Is 9 a good default for ZSTD now that we've switched to it as default? Re: comment above with different ranges of values for LZ4 and ZSTD.